etc/scripts/github_release.py

```diff
@@ -27,127 +27,138 @@
 from __future__ import print_function
 
 import argparse
-from fnmatch import fnmatchcase
+from pathlib import Path
 import os
-from subprocess import run
 import sys
 
+from github_release_retry import github_release_retry as grr
+
 from commoncode.fileutils import resource_iter
 
-python_version = str(sys.version_info[0]) + str(sys.version_info[1])
```

Comment:
Why remove this section ,it will help you filter the python 3 files if your directory contains multiple python deps.